webapp/src/pages/_app.tsx

```diff
@@ -4,8 +4,8 @@ import { Provider } from 'react-redux';
 import { ChakraProvider, theme } from '@chakra-ui/react';
 import { Authorized } from 'dogma/features/auth/Authorized';
 import { NextPage } from 'next';
-import { ReactElement, ReactNode } from 'react';
-import { useRouter } from 'next/router';
+import { ReactElement, ReactNode, useEffect } from 'react';
+import Router, { useRouter } from 'next/router';
 import { Layout } from 'dogma/common/components/Layout';
```

Comment:
We can also get `router` from `router = useRouter()`, they are actually similar. I think the difference is the non global `router` will return a different path when we call `router.asPath`.